docs/reference/release-notes/highlights.asciidoc

```diff
@@ -15,6 +15,32 @@ For detailed information about this release, see the <<es-release-notes>> and
 // should be featured in the Stack Installation and Upgrade Guide:
 
 // tag::notable-highlights[] 
+[discrete]
+=== Security is enabled and configured by default
+
+Running {es} without security leaves your cluster exposed to anyone who can send
+network traffic to {es}. In previous versions, you had to enable the {es}
+security features and then configure password protection and Transport Layer
+Security (TLS).
+
+Starting in {es} 8.0, security is
+{ref}/configuring-stack-security.html[enabled and configured by default] when
+you start {es} for the first time. We also generate an enrollment token at
```

Comment:
I think it would be clearer if we don't make the distinction between the tokens in such a way (For instance, we generate an enrollment token for nodes in docker, but reading this makes you think that you have to do something additional in that case too ). How about we say something more generic similar to:

"We generate enrollment tokens so that you can quickly and easily connect a {kib} instance 
or additional nodes to your secured {es} cluster without having to generate security certificates or update
`yml` configuration files. 
Just use the enrollment token when starting new nodes or {kib} instances and the elastic stack handles all of the security configuration for you. Out of the box, you’ll get authentication, authorization, network encryption 
with TLS for internode communications and communications between {es} and {kib}.
You can always create new enrollment tokens for {es} nodes and {kib} instances using the {ref}/create-enrollment-token.html[`elasticsearch-create-enrollment-token`] tool."

WDYT ?

Reply:
I like your suggestions @jkakavas -- I'll incorporate them in my next push.